nuget.config

```diff
@@ -1,13 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
 <configuration>
 	<config>
 		<clear />
 	</config>
-	<packageSourceMapping>
-		<packageSource key="nuget.org">
-			<package pattern="*" />
-		</packageSource>
-		<packageSource key="https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-Labs/nuget/v3/index.json">
-			<package pattern="CommunityToolkit.Labs.*" />
-		</packageSource>
-	</packageSourceMapping>
+	<packageSources>
+		<clear />
+		<!-- this feed probably won't work -->
+		<add key="TerminalDependencies" value="https://pkgs.dev.azure.com/ms/terminal/_packaging/TerminalDependencies/nuget/v3/index.json" />
```

Comment:
how would this fix work for general community?

Reply:
Just how it does for the `Terminal` project :) The above feed has anonymous read-only access.  Folks who want to add a new package will need to 1) temporarily add nuget.org while they are coding and 2) submit a PR w/o nuget.org.  The PR will kickoff a build which will have the access to "pull-through" packages from nuget.org into the feed.

Reply:
Hi @johnterickson , The PR was failing CI before and it's also failing the CI now after I've tried to merge main in. The reason being that it can't run a script because it's missing some dependencies from nuget. I think this will make a much worse experience for the community. I don't think labs is needed anymore, though, I think we can remove the labs dependency since what we used from there it's been included into the community toolkit already.
Regarding changing of feeds, the files used by the Dart CI for release are the ones in ".pipelines/packages.config" and ".pipelines/release-nuget.config". Does it make more sense to change in those instead?